query-engine/prisma-models/src/selection_result.rs

```diff
@@ -0,0 +1,172 @@
+use crate::{DomainError, FieldSelection, PrismaValue, ScalarFieldRef, SelectedField};
+use std::convert::TryFrom;
+
+/// Represents a set of results.
+#[derive(Default, Debug, Clone, PartialEq, Eq, Hash)]
+// Name candidates:
+// - ReturnValues
+// - PrismaValues
+// - Values
+// - ResultValues
+// - SelectionResult << This would remove SelectedField and reuse the SelectedField concept.
```

Comment:
> This would remove SelectedField and reuse the SelectedField concept

Not sure if there's a typo here but I'm unable to deduce what it would remove 🙈

Reply:
Ignore, this was just notes during prototyping. Deleted.